Automattic/pocket-casts-ios
# Transcripts filter content

Refs #1848 

This PR  improves the display of the transcript by applying filters to the content, in order to remove Speaker information and better layout the text depending of the existence of paragraphs.

https://github.com/Automattic/pocket-casts-ios/assets/651601/e6b0f8c4-5a56-4431-a117-54bec639630e

## To test

- Start the app
- Ensure that you have the `transcripts` and `newShowNotesEndpoint` FF enabled
- Start playing an episode with Transcript support. Ex: Any episode The Audacity to Podcast
- Open the full screen player
- Tap on the transcript tabs
- See that the layout of text, is respecting paragraphs and no "Speaker" entries or other xml kind of tags `<>` are present on on the text.

## Checklist
